### PR TITLE
S&F | Sessions-refactor

### PR DESCRIPTION
**Pull request by LinX64**

### CODE REVIEW

**Comment by LinX64** [comment]
Couldn't see the sessions without UTC, that's why I made it similar to IOS.

**Comment by LinX64** [comment]
[No sessions displayed west of specific longitude](https://trello.com/c/GyHf8lcW/1682-no-sessions-displayed-west-of-specific-longitude)

File: app/src/main/java/pl/llp/aircasting/util/CalendarUtils.kt [hunk 1]
```diff
@@ -13,15 +13,19 @@ class CalendarUtils {
...
-         * Our time zone: Sunday, August 1, 2021 2:00:00 AM GMT+02:00 DST
+         * EXAMPLE: if the current day is Aug 2:
+         * @return "1627862400"
+         * Assuming that this timestamp is in seconds:
+         * GMT: Monday, August 2, 2021 12:00:00 AM
+         * Your time zone: Monday, August 2, 2021 2:00:00 AM GMT+02:00 DST
+         * Relative: A year ago
          **/
         fun getStartOfTodayMillisFromYearAgo(): Long {
             val today = calendar()
```
Comment (xameYeoh) [inline app/src/main/java/pl/llp/aircasting/util/CalendarUtils.kt:26]: I guess now we should name in getStartOfTodaySecondsFromYearAgo and getEndOfTodaySeconds

**Review by xameYeoh — APPROVED** [review]

**Comment by LinX64** [comment]
[S&F > map: sessions list scrolls a long time](https://trello.com/c/PJ0qXXKB/1676-sf-map-sessions-list-scrolls-a-long-time)

**Pull request merged**